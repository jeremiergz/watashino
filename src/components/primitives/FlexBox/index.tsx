import Box, { BoxProps } from 'components/primitives/Box';
import React from 'react';

const FlexBox: React.FC<BoxProps> = ({ children, ...rest }) => {
  return (
    <Box display="flex" {...rest}>
      {children}
    </Box>
  );
};

FlexBox.displayName = 'FlexBox';

export type FlexBoxProps = BoxProps;
export default FlexBox;
