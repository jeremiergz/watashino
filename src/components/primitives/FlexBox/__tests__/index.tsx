import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import FlexBox from '..';

describe('components/primitives/FlexBox Test Suite', () => {
  it('matches snapshot', () => {
    const jsx = <FlexBox>FlexBox Component</FlexBox>;
    expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
  });
});
