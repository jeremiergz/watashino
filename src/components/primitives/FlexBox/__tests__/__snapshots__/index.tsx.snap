// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/primitives/FlexBox Test Suite matches snapshot 1`] = `
<div
  className="sc-AxjAm fzFkQq"
  display="flex"
>
  FlexBox Component
</div>
`;
